### ISSUE TITLE
Event manager tests broken

### ISSUE THREAD

**Issue by liayoo** [ainblockchain/ain-js#59]
Event manager tests are broken even with the changes in #58.
Ran the test with ain-blockchain on develop at commit d69cbd2f711080ea9a485d6f8319083dcf375eb3.

```
 FAIL  __tests__/event_manager.test.ts (181.367s)
  ● Console

    console.info node_modules/eccrypto/index.js:23
      secp256k1 unavailable, reverting to browser version
    console.error node_modules/jest-jasmine2/build/jasmine/Env.js:289
      Unhandled error
    console.error node_modules/jest-jasmine2/build/jasmine/Env.js:290
      TypeError: Cannot read properties of null (reading 'url')
          at EventChannelClient.<anonymous> (/Users/liayoo/comcom/ain-js/src/event-manager/event-channel-client.ts:34:43)
          at step (/Users/liayoo/comcom/ain-js/src/event-manager/event-channel-client.ts:33:23)
          at Object.next (/Users/liayoo/comcom/ain-js/src/event-manager/event-channel-client.ts:14:53)
          at fulfilled (/Users/liayoo/comcom/ain-js/src/event-manager/event-channel-client.ts:5:58)
          at processTicksAndRejections (node:internal/process/task_queues:96:5)

  ● Event Handler › Subscribe to BLOCK_FINALIZED

    Timeout - Async callback was not invoked within the 180000ms timeout specified by jest.setTimeout.Error: Timeout - Async callback was not invoked within the 180000ms timeout specified by jest.setTimeout.

      at mapper (node_modules/jest-jasmine2/build/queueRunner.js:25:45)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)

  ● Event Handler › Subscribe to BLOCK_FINALIZED

    Event channel is not connected! You must call ain.eh.connect() before using subscribe()

      44 |       dataCallback?: (data: any) => void, errorCallback?: (error: any) => void): string {
      45 |     if (!this._eventChannelClient.isConnected) {
    > 46 |       throw Error(`Event channel is not connected! You must call ain.eh.connect() before using subscribe()`);
         |             ^
      47 |     }
      48 |     const filter = this._eventCallbackManager.createFilter(eventTypeStr, config);
      49 |     this._eventChannelClient.registerFilter(filter);

      at EventManager.Object.<anonymous>.EventManager.subscribe (src/event-manager/index.ts:46:13)
      at Suite.<anonymous> (__tests__/event_manager.test.ts:21:12)
      at step (__tests__/event_manager.test.ts:33:23)
      at Object.next (__tests__/event_manager.test.ts:14:53)
      at __tests__/event_manager.test.ts:8:71
      at Object.<anonymous>.__awaiter (__tests__/event_manager.test.ts:4:12)
      at Object.<anonymous> (__tests__/event_manager.test.ts:19:38)

Test Suites: 1 failed, 2 passed, 3 total
Tests:       1 failed, 55 passed, 56 total
Snapshots:   15 passed, 15 total
Time:        181.74s, estimated 188s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by cshcomcom** [comment]
Thanks for report! I'll fix it with #56 PR

**Comment by cshcomcom** [comment]
To pass the event manager test, we need to do some preliminary work, so I added it to the readme. Thanks!
